azurerm/resource_arm_dns_zone.go

```diff
@@ -51,87 +51,78 @@ func resourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"etag": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
 			"tags": tagsSchema(),
 		},
 	}
 }
 
 func resourceArmDnsZoneCreate(d *schema.ResourceData, meta interface{}) error {
-	client := meta.(*ArmClient)
-	rivieraClient := client.rivieraClient
+	zonesClient := meta.(*ArmClient).zonesClient
 
-	tags := d.Get("tags").(map[string]interface{})
-	expandedTags := expandTags(tags)
-
-	createRequest := rivieraClient.NewRequest()
-	createRequest.Command = &dns.CreateDNSZone{
-		Name:              d.Get("name").(string),
-		Location:          "global",
-		ResourceGroupName: d.Get("resource_group_name").(string),
-		Tags:              *expandedTags,
-	}
+	zoneName := d.Get("name").(string)
+	resGroup := d.Get("resource_group_name").(string)
+	eTag := d.Get("etag").(string)
+	location := "global"
 
-	createResponse, err := createRequest.Execute()
-	if err != nil {
-		return fmt.Errorf("Error creating DNS Zone: %s", err)
-	}
-	if !createResponse.IsSuccessful() {
-		return fmt.Errorf("Error creating DNS Zone: %s", createResponse.Error)
-	}
+	tags := d.Get("tags").(map[string]interface{})
+	metadata := expandTags(tags)
 
-	readRequest := rivieraClient.NewRequest()
-	readRequest.Command = &dns.GetDNSZone{
-		Name:              d.Get("name").(string),
-		ResourceGroupName: d.Get("resource_group_name").(string),
+	parameters := dns.Zone{
+		Name:     &zoneName,
+		Location: &location,
+		Tags:     metadata,
```

Comment:
same question as in the other PR - is `metadata` the same as `tags` in the previous API? the documentation is unclear